### PR TITLE
virtuoso 7.2.6

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/virtuoso.rb [hunk 1]
```diff
@@ -38,7 +37,7 @@ class Virtuoso < Formula
 
   # Support OpenSSL 1.1
   patch do
-    url "https://sources.debian.org/data/main/v/virtuoso-opensource/7.2.5.1+dfsg-2/debian/patches/ssl1.1.patch"
+    url "https://sources.debian.org/data/main/v/virtuoso-opensource/7.2.6.1+dfsg-2/debian/patches/ssl1.1.patch"
```
Comment (cho-m) [inline Formula/virtuoso.rb:40]: `curl: (22) The requested URL returned error: 404 `

For both previous and updated URLs.

Comment (cho-m) [reply Formula/virtuoso.rb]: Maybe just delete the patch?
7.2.6+ has commits like:
- **Added initial support for OpenSSL 1.1.1** https://github.com/openlink/virtuoso-opensource/commit/713fa25b14457aa5127fac071830a2d20f4f968c
- **Removed support for OpenSSL 0.9.7 and earlier** https://github.com/openlink/virtuoso-opensource/commit/88c4eed7c4549f3162be4057b184b79cf231068d

File: Formula/virtuoso.rb [hunk 2]
```diff
@@ -36,19 +32,6 @@ class Virtuoso < Formula
...
-  # TLS 1.3 compile error patch.
-  # This also updates the default TLS protocols to allow TLS 1.3.
-  patch do
-    url "https://github.com/openlink/virtuoso-opensource/commit/67e09939cf62dc753feca8381396346f6d3d4a06.patch?full_index=1"
-    sha256 "485f54e4c79d4e1e8b30c4900e5c10ae77bded3928f187e7e2e960d345ca5378"
-  end
-
   def install
     # We patched configure.ac on stable so need to rerun the autogen.
     system "./autogen.sh"
```
Comment (cho-m) [inline Formula/virtuoso.rb:37]: Shouldn't be needed for stable anymore. We can restore head.
```suggestion
    system "./autogen.sh" if build.head?
```

Along with
```diff
@@ -13,11 +12,16 @@ class Virtuoso < Formula
     sha256 cellar: :any, high_sierra: "3abcc2f1444324d675af9014ac20555124c875d7e9a4ba9b021fd1ad7c570845"
   end

-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/openlink/virtuoso-opensource.git", branch: "develop/7"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   # If gawk isn't found, make fails deep into the process.
   depends_on "gawk" => :build
-  depends_on "libtool" => :build
   depends_on "openssl@1.1"
```

File: Formula/virtuoso.rb [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
-  url "https://github.com/openlink/virtuoso-opensource/releases/download/v7.2.5.1/virtuoso-opensource-7.2.5.tar.gz"
-  # Upstream pushed a hot-fix retag of 7.2.5 as 7.2.5.1.
-  # This explicit version should be safe to remove next release.
-  version "7.2.5.1"
-  sha256 "826477d25a8493a68064919873fb4da4823ebe09537c04ff4d26ba49d9543d64"
+  url "https://github.com/openlink/virtuoso-opensource/releases/download/v7.2.6.1/virtuoso-opensource-7.2.6.tar.gz"
+  sha256 "38fd3c037aef62fcc7c28de5c0d6c2577d4bb19809e71421fc42093ed4d1c753"
   license "GPL-2.0-only"
-  revision 1
   # HEAD is disabled as the below, required patches are not compatible.
```
Comment (cho-m) [inline Formula/virtuoso.rb:7]: Also remove if restoring head.
```suggestion
```

**Comment by chenrui333** [comment]
@cho-m all nice suggestions! 💯  I will update the PR shortly.

**Comment by chenrui333** [comment]
restored the changes made in #46885

**Review by cho-m — APPROVED** [review]

**Pull request closed without merging**